jib_triage_party.yaml

```diff
@@ -0,0 +1,754 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+settings:
+  name: jib
+  min_similarity: 0.8
+  repos:
+    - https://github.com/GoogleContainerTools/jib
```

Comment:
The only strange thing I feel about this is that the repos unrelated to jib are included in here. Should this config not reside in github?

Reply:
hm yeah I feel a little strange about this as well.